cmd/iso_package/main.go

```diff
@@ -44,6 +43,34 @@ func init() {
 			Usage: "Arch to compile to, possible values: x86, aarch. Default x86",
 			Value: "x86",
 		},
+
+		&cli.StringFlag{
+			Name:  "upload-target",
+			Usage: "target to upload the s3 file to, like s3://mybucket/myfile.iso",
+		},
+		&cli.StringFlag{
+			Name:    "aws_access_key_id",
+			Usage:   "AWS_ACCESS_KEY_ID to store the iso on s3",
+			EnvVars: []string{"AWS_ACCESS_KEY_ID"},
+		},
+
+		&cli.StringFlag{
+			Name:    "AWS_SECRET_ACCESS_KEY",
+			Usage:   "AWS_SECRET_ACCESS_KEY to store the iso on s3",
+			EnvVars: []string{"AWS_SECRET_ACCESS_KEY"},
+		},
+
+		&cli.StringFlag{
+			Name:    "AWS_DEFAULT_REGION",
+			Usage:   "AWS_DEFAULT_REGION to store the iso on s3",
+			EnvVars: []string{"AWS_DEFAULT_REGION"},
+		},
+
+		&cli.StringFlag{
```

Comment:
I made a mistake when I categorized the parameters. Actually, the Endpoint should be optional since it is used only for Generic S3 services and is not used when uploading to AWS